pipeline/src/client.py

```diff
@@ -89,6 +91,8 @@ def main(self, read_env_data=False):
             ai = ArgInterface()
             return ai.query_with_args(self, sys.argv[1:])
 
+        self.init_flag_dict()
+
```

Comment:
I think this should be in the constructor as this seems to be a very initialization-y kinda thing.

Reply:
I did not put it there because in the event daily or a range query is getting used from ArgInterface, this dict does not need to be loaded from the db, or initialized at all.